spec/builtin-read.test.sh

```diff
@@ -773,3 +773,99 @@ status=1
 status=0
 ## END
 ## N-I dash/ash/mksh/zsh stdout-json: ""
+
+#### read -n 0
+case $SH in (zsh) exit 99;; esac  # read -n not implemented
```

Comment:
minor style preference: I usually write `zsh)` instead of `(zsh)` now

For awhile I used `(zsh)`, but now YSH uses the symmetrical style for expressions, like

`case (x) { (42) { echo number } }`

So I try to keep it distinct

Reply:
I just copied the existing ones in the same file. I fixed it ([`592aaf4..77d499c`](https://github.com/oils-for-unix/oils/compare/592aaf46067f89840acc640a7faff703e83336fd..77d499caef4680c3abe17100de6678562e245589)). I'll later submit a PR to fix existing ones.